lib/js/dom.nim

```diff
@@ -1136,7 +1144,7 @@ proc createAttribute*(d: Document, identifier: cstring): Node
 proc getElementsByName*(d: Document, name: cstring): seq[Element]
 proc getElementsByTagName*(d: Document, name: cstring): seq[Element]
 proc getElementsByClassName*(d: Document, name: cstring): seq[Element]
-proc getSelection*(d: Document): cstring
+proc getSelection*(d: Document): Selection
```

Comment:
This is a breaking change, add a `converter` which implicitly converts `Selection` to `cstring`.

Reply:
converters are a little bit evil, can we find another way? I'd rather have a different overload and deprecate previous

Reply:
Should be used or not, it's a breaking change, please introduce ``proc getSelectionEnum`` or similar.

Reply:
I have reverted getSelection to the previous cstring behavior. No changes to the API.

Reply:
how about turning code into:
```nim
proc getSelection*(d: Document): cstring {.importc: "getSelection", deprecated: "incompatible across browsers, use instead XYZ".}
proc getSelectionAsObject*(d: Document): Selection {.importc: "getSelection", deprecated: "incompatible across browsers, use instead XYZ".}
```

Reply:
Yeah, I want to do some thing like that in another commit.